target/linux/realtek/files-5.15/drivers/net/dsa/rtl83xx/rtl930x.c

```diff
@@ -2396,23 +2396,62 @@ static void rtl930x_led_init(struct rtl838x_switch_priv *priv)
 		return;
 	}
 
+	for (int set = 0; set < 4; set++) {
+		char set_name[16] = {0};
+		u32 set_config[4];
+		int leds_in_this_set = 0;
+
+#define RTL930X_LED_SETX_0_CTRL(x) RTL930X_LED_SET0_0_CTRL - (x*8)
+#define RTL930X_LED_SETX_1_CTRL(x) RTL930X_LED_SETX_0_CTRL(x) - 4
+
+// get register for given set and led in the set
+#define RTL930X_LED_SETX_LEDY(x,y) RTL930X_LED_SETX_0_CTRL(x) - 4*(y/2)
+
+// get shift for given led in any set
+#define RTL930X_LED_SET_LEDX_SHIFT(x) (16*(x%2))
```

Comment:
This gives the impression that these definitions are scoped, when they aren't. Just place them at the top of the file with the other `#define`s. It would also be prudent to enclose all the definitions in parentheses, like you did for `RTL930X_LED_SET_LEDX_SHIFT(x)`. And as a final nit, kernel style is to add spaces around operators: `(16 * (x % 2))`

Reply:
done